resolver/resolver.go

```diff
@@ -45,6 +50,10 @@
```

Comment:
The other option here is to simply truncate the tags to be the maximum amount, since the spec states that nodes are not required to honour tags.

Reply:
I think there would be less trouble for developers if we explicitly returned an error